packages/config/eslint/index.js

```diff
@@ -29,6 +29,10 @@ const config = {
         disallowTypeAnnotations: false,
       },
     ],
+    "@typescript-eslint/no-misused-promises": [
+      2,
```

Comment:
should this be a number, or `warn` / `error` / etc?

Reply:
2 == 'error' - i can change it for verbosity